libraryBuilds/com.fasterxml.jackson.core/com.fasterxml.jackson.core-jackson-databind/src/custom-test/java/com/fasterxml/jackson/custom/GenericsTest.java

```diff
@@ -0,0 +1,193 @@
+package com.fasterxml.jackson.custom;
```

Comment:
I don't understand this directory 'src/custom-test' and this file.
Is it not a duplicate of the other test in src/test?